test/fuzz/transformation_swap_commutable_operands_test.cpp

```diff
@@ -0,0 +1,360 @@
+// Copyright (c) 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "source/fuzz/transformation_swap_commutable_operands.h"
+#include "source/fuzz/instruction_descriptor.h"
+#include "test/fuzz/fuzz_test_util.h"
+
+namespace spvtools {
+namespace fuzz {
+namespace {
+
+TEST(TransformationSwapCommutableOperandsTest, IsApplicableTest) {
```

Comment:
Please update this test with some cases that fail due to the described instruction not existing, including:

- the base instruction id not existing
- there being no instruction with the desired opcode after the base instruction id
- there being an instruction with the desired opcode after the base instruction id, but the skip count associated with the instruction descriptor being so high.

Reply:
Done.